### PR TITLE
User story 14330 schedule trigger

### PR DESCRIPTION
**Pull request by rakesh-satelli**
Added the scheduled trigger.
This PR is as part of the User-Story: 14330 [AB14330](https://dev.azure.com/3M-Bluebird/AzurePlatform/_backlogs/backlog/Odin/Stories/?workitem=14330)

### CODE REVIEW

File: pipelines/cd/infra.yaml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 trigger: none
 pr: none
+schedules:
+  - cron: 0 5 * * *
```
Comment (kyleestes) [inline pipelines/cd/infra.yaml:4]: You  might need to adjust this cron line to reflect the time we dicussed on the call (morning in India, evening in America). Right now, I think it is set for midnight in America.

Comment (rakesh-satelli) [reply pipelines/cd/infra.yaml:4]: Hi @kyleestes 
Can I go ahead and change to 0 2 * * *. Which will be equivalent to 9.pm CST and 7:30 am IST

File: pipelines/cd/infra.yaml [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+    displayName: Daily midnight (CST) build
```
Comment (joe-bethke) [inline pipelines/cd/infra.yaml:5]: we'll also want to change this displayName as well if we're no longer running at midnight right?

Comment (rakesh-satelli) [reply pipelines/cd/infra.yaml]: Yes Joe, updated the display name and raised another PR for this

**Pull request closed without merging**